ports/raspberrypi/boards/lilygo_t_display_rp2040/board.c

```diff
@@ -99,9 +99,9 @@ static void display_init(void) {
         bus,
         240,            // width (after rotation)
         135,            // height (after rotation)
-        53,             // column start
-        40,             // row start
-        90,             // rotation
+        40,             // column start
+        53,             // row start
+        0,              // rotation
```

Comment:
Configurations which have a "0 degree" rotation of the displayio display are almost always preferable.